### DISCUSSION TITLE
How to install SSL certification on Netbox ?

### DISCUSSION THREAD

**Discussion by Jason-MK** [netbox-community/netbox#7330] [General]
Hello , 

I have an official SSL CERT and I really need to know how to install SSL certification on Netbox ? 

Best Regards

**Comment by domainzero** [comment]
The easiest way is probably to use a reverse proxy like nginx or something and terminate SSL there, but it looks like gunicorn allows you to use a certificate now. There are steps outlined in [the documentation](https://netbox.readthedocs.io/en/stable/installation/5-http-server/) for using a reverse proxy with SSL.
